synapse/replication/tcp/streams/_base.py

```diff
@@ -463,18 +463,66 @@ class DeviceListsStream(Stream):
     @attr.s(slots=True, frozen=True, auto_attribs=True)
     class DeviceListsStreamRow:
         entity: str
+        is_signature: bool
```

Comment:
Can we add a comment, borrowed from the old `UserSignatureStream` docstring?
```suggestion
        # Indicates that a user has signed their own device with their user-signing key
        is_signature: bool
```

Reply:
Whoops, thanks.